content/en/developers/authorization/oauth2_endpoints.md

```diff
@@ -0,0 +1,234 @@
+
+#### Example Request 
+
+Use this cURL command to make an access token request:
+
+```
+curl -X POST \
+    -d "grant_type=authorization_code&client_id=$CLIENT_ID
+    client_secret=$CLIENT_SECRET&redirect_uri=$REDIRECT_URI
+    code_verifier=$CODE_VERIFIER&code=$CODE" \
+    "https://api.datadoghq.com/oauth2/v1/token"
+```
+
+#### Success Response
+
+If the access token request is valid and authorized, the [token response][7] returns a `200 OK` status code with the access token and refresh token contained in the body of the HTTP response. 
+
+#### Error Response
+
+Failed requests made to token endpoints should be handled by the application, such as redirecting users to an appropriate error page on the application. 
+
+If a confidential client with an issued client secret makes a token request without providing the `client_secret` parameter, a `401 Unauthorized` status code is returned.
+
+If a token request fails for other reasons, such as a malformed request or an invalid authorization code, a `400 Bad Request` status code (unless specified otherwise) is returned with an [`error`][8] parameter. 
+
+## Revoke tokens 
+
+### `POST /oauth2/v1/revoke`
+
+#### Overview
+
+Users can revoke access or refresh tokens at any time. When revoked, tokens can no longer be used to access Datadog APIs. To revoke a given token, your application makes a POST request to Datadog’s [token revocation endpoint][9]. 
+
+#### Request
+The [revocation request][10] is made with the following parameters in the **body** of the `HTTP POST` request with the `application/x-www-form-urlencoded` format:
+
+| HTTP Body Parameter          | Description                                                                                                              |
+|------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| `client_id`                    | The client ID of your OAuth2 client.                                                                                      |
+| `client_secret` (if issued)                    | The client secret of your OAuth2 private client.                                                                                       |
+| `token`                        | The token string to be revoked.                                                                                           |
+| `token_type_hint` (optional)   | A hint about the token type to be revoked to help optimize token lookup. For example, `access_token` or `refresh_token`.  |
+
+#### Code Example 
+
+Use this cURL command to make a revocation request:
+
+```
+curl -X POST \
+    -H "Authorization: Bearer $ACCESS_TOKEN" \
+    -d 'client_id=$CLIENT_ID&token=$TOKEN_TO_REVOKE' \
```

Comment:
```suggestion
    -d 'client_id=$CLIENT_ID&client_secret=$CLIENT_SECRET&token=$TOKEN_TO_REVOKE' \
```

I think we should add `CLIENT_SECRET` here

Reply:
Will do